runtime/packages/models/assembly/transaction-models/transaction-error.ts

```diff
@@ -0,0 +1,26 @@
+export class TransactionError{
```

Comment:
This is very similar to the Apply Extrisic result file. We can discuss how to mitigate the code duplication

Reply:
We will create `ResponseCodes` class with all of them